tutorial/source/air.ipynb

```diff
@@ -41,7 +41,7 @@
     "import numpy as np\n",
     "\n",
     "smoke_test = ('CI' in os.environ)\n",
-    "assert pyro.__version__.startswith('1.6.0')"
+    "assert pyro.__version__.startswith("1.7.0")"
```

Comment:
You might need [this](https://github.com/pyro-ppl/numpyro/pull/1082/files#diff-b10936dd06227522f59969cfad4e8d73869e27a5625f5a9aacf1ade10683c889L25) to bump versions for the notebooks.